<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtQQServer</class>
 <widget class="QDialog" name="QtQQServer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>999</width>
    <height>502</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>999</width>
    <height>502</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>999</width>
    <height>502</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>QtQQServer</string>
  </property>
  <widget class="QScrollArea" name="scrollArea">
   <property name="geometry">
    <rect>
     <x>1</x>
     <y>1</y>
     <width>700</width>
     <height>500</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>700</width>
     <height>500</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>700</width>
     <height>500</height>
    </size>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>698</width>
      <height>498</height>
     </rect>
    </property>
    <widget class="QTableWidget" name="tableWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>700</width>
       <height>500</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>702</x>
     <y>1</y>
     <width>296</width>
     <height>500</height>
    </rect>
   </property>
   <widget class="QLabel" name="imgLabel">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>301</width>
      <height>211</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
    <property name="pixmap">
     <pixmap>Resources/img/fly1.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>210</y>
      <width>291</width>
      <height>281</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="Line" name="line">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>170</y>
       <width>291</width>
       <height>3</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QComboBox" name="employeeDepBox">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>20</y>
       <width>113</width>
       <height>28</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>人事部</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>研发部</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>市场部</string>
      </property>
     </item>
    </widget>
    <widget class="QLineEdit" name="nameLineEdit">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>60</y>
       <width>113</width>
       <height>28</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>请输入员工姓名</string>
     </property>
    </widget>
    <widget class="QPushButton" name="selectPictureBtn">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>100</y>
       <width>31</width>
       <height>32</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
    <widget class="QPushButton" name="addBtn">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>140</y>
       <width>113</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>新增员工</string>
     </property>
    </widget>
    <widget class="QLabel" name="headLabel">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>100</y>
       <width>41</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>头像</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QWidget" name="">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>10</y>
       <width>61</width>
       <height>131</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>员工部门:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>员工姓名:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>员工头像:</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>190</y>
       <width>271</width>
       <height>91</height>
      </rect>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <property name="labelAlignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
      <property name="horizontalSpacing">
       <number>15</number>
      </property>
      <property name="verticalSpacing">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <widget class="QComboBox" name="departmentBox">
        <item>
         <property name="text">
          <string>公司群</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>人事部</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>研发部</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>市场部</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="queryDepartmentBtn">
        <property name="minimumSize">
         <size>
          <width>113</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>113</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>查询员工</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLineEdit" name="queryIDLineEdit">
        <property name="placeholderText">
         <string>请输入员工 ID</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="queryIDBtn">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>113</width>
          <height>29</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>113</width>
          <height>29</height>
         </size>
        </property>
        <property name="text">
         <string>查询员工</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLineEdit" name="logoutIDLineEdit">
        <property name="placeholderText">
         <string>请输入员工 ID</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QPushButton" name="logoutBtn">
        <property name="minimumSize">
         <size>
          <width>113</width>
          <height>29</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>113</width>
          <height>29</height>
         </size>
        </property>
        <property name="text">
         <string>注销员工</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
